edit.html

```diff
@@ -50,9 +55,14 @@
     <script src="vendor/codemirror/addon/hint/show-hint.js"></script>
     <script src="vendor/codemirror/addon/hint/css-hint.js"></script>
 
+    <script src="vendor/codemirror/addon/mode/loadmode.js"></script>
+
     <script src="vendor/codemirror/keymap/sublime.js"></script>
     <script src="vendor/codemirror/keymap/emacs.js"></script>
     <script src="vendor/codemirror/keymap/vim.js"></script>
+
+    <script src="/vendor-overwrites/codemirror/codemirror-default.js"></script>
+    <link rel="stylesheet" href="/vendor-overwrites/codemirror/codemirror-default.css">
```

Comment:
put it in `edit/`

>Can we put them in a codemirror/ folder? 

No, because the name is too amiguous.

>Or put codemirror-default.js to js/ and codemirror-default.css to css/.

No, because those are for more global stuff.